static/js/public/snap-details/map.js

```diff
@@ -0,0 +1,121 @@
+/* global d3, topojson */
+
+export default function renderMap(el, snapData, isPublic) {
```

Comment:
I'm not a big fan of bool params hanging in the end, as it makes it hard to deduce their meaning from the calls:
`snapcraft.public.map('#js-snap-map', {{ countries|tojson }}, true);` - what is `true`?

Also in this case we are interested in the opposite of `isPublic` - even in the code below we are checking `if (!isPublic)`, so it seems that `isAuthenticated` or `isPublisher` would make more sense.

So a simpler suggestion is to change to `isPublisher` that defaults to false/undefined and has to be set to true if user can see more data in the tooltip.

On the other hand - if the user is not supposed to see the user counts they should not be in data, so we don't really need `isPublic` or `isPublisher`, because it's the data itself that will define what to show in the tooltip.

Reply:
Yeah I had it like that before (using the data) however @lyubomir-popov brought up that not showing the 'x daily active users' made him think it was the public map, so I added this bool to ensure that '0 daily active users' is displayed... I also had it that way round initially, being a `isPublisher` but I swapped it for a reason I don't remember.... Will switch it back though as I agree.

Reply:
@Lukewh Thanks for update.

Still, I think that existence of `number_of_users` (even if it's zero) can trigger showing publisher tooltip while nonexistence of it (when it's undefined, not 0) will show public tooltip.

Reply:
Yeah I thought this after doing the last commit. Will update again, thanks @bartaz 💟

Reply:
pushed @bartaz